qemu/tests/cfg/qemu_guest_agent.cfg

```
@@ -15,13 +15,20 @@
     setsebool_cmd = "setsebool virt_qemu_ga_read_nonsecurity_files %s"
     getsebool_cmd = "getsebool -a | grep virt_qemu_ga_read_nonsecurity_files |awk '{print$3}'"
     backup_file = /etc/sysconfig/qemu-ga-bk
-    black_list_backup = /bin/cp -f /etc/sysconfig/qemu-ga ${backup_file}
+    black_file_backup = /bin/cp -f /etc/sysconfig/qemu-ga ${backup_file}
     recovery_black_list = mv -f ${backup_file} /etc/sysconfig/qemu-ga
-    black_list_change_cmd = "sed -i 's/%s//g' /etc/sysconfig/qemu-ga"
-    black_list_spec = BLOCK_RPCS
-    Host_RHEL.m7, Host_RHEL.m8, Host_RHEL.m9.u0, Host_RHEL.m9.u1:
-        black_list_spec = BLACKLIST_RPC
+    black_list_spec = allow-rpcs
+    black_list_change_cmd = sed -i 's/${black_list_spec}.*/${black_list_spec}=%s"/g' /etc/sysconfig/qemu-ga
     black_list_check_cmd = "grep -nr '${black_list_spec}=.*%s' /etc/sysconfig/qemu-ga"
+    black_list_new = "guest-file-open,guest-file-close,guest-file-read,guest-file-write,guest-file-seek,guest-file-flush"
+    Host_RHEL.m7, Host_RHEL.m8, Host_RHEL.m9.u0, Host_RHEL.m9.u1:
+        black_list_spec_old = BLACKLIST_RPC
+        black_list_change_cmd = "sed -i 's/%s//g' /etc/sysconfig/qemu-ga"
+        black_list_check_cmd = "grep -nr '${black_list_spec_old}=.*%s' /etc/sysconfig/qemu-ga"
+    Host_RHEL.m9.u2, Host_RHEL.m9.u3:
+        black_list_spec_old = BLOCK_RPCS
+        black_list_change_cmd = "sed -i 's/%s//g' /etc/sysconfig/qemu-ga"
+        black_list_check_cmd = "grep -nr '${black_list_spec_old}=.*%s' /etc/sysconfig/qemu-ga"
```

I don't see why we have to introduce `black_list_spec_old`, can below config work as you expected?
```suggestion
    black_list_spec = allow-rpcs
    black_list_change_cmd = sed -i 's/${black_list_spec}.*/${black_list_spec}=%s"/g' /etc/sysconfig/qemu-ga
    black_list_new = "guest-file-open,guest-file-close,guest-file-read,guest-file-write,guest-file-seek,guest-file-flush"
    Host_RHEL.m7, Host_RHEL.m8, Host_RHEL.m9.u0, Host_RHEL.m9.u1:
        black_list_spec = BLACKLIST_RPC
        black_list_change_cmd = "sed -i 's/%s//g' /etc/sysconfig/qemu-ga"
    Host_RHEL.m9.u2, Host_RHEL.m9.u3:
        black_list_spec = BLOCK_RPCS
        black_list_change_cmd = "sed -i 's/%s//g' /etc/sysconfig/qemu-ga"
    black_list_check_cmd = "grep -nr '${black_list_spec}=.*%s' /etc/sysconfig/qemu-ga"
```

---

Yeah, it makes sense. changed.